src/components/navBar/navBar.module.scss

```diff
@@ -1,35 +1,217 @@
-$offset: 10px;
-$thickness: 3px;
+.wrapper {
+  font: 700 16px "Roboto", sans-serif;
+  margin-bottom: 15px;
+}
 
-.navBar {
-  display: flex;
-  width: 100%;
+.navBarLogin,
+.homeTab {
+  display: none;
+}
+
+.navBar li a {
+  margin: 10px;
+  display: block;
+  border-radius: 0.5rem;
+  color: #ffffff;
+  text-align: center;
+  padding: 19px 16px;
+  text-decoration: none;
+}
+
+.navBar li {
```

Comment:
Do not use `float`. Use `flex`.

`UL` -> `display: flex` and then play with the spacing.

Reply:
Done